thirdeye/thirdeye-frontend/app/pods/services/query-cache/service.js

```diff
@@ -0,0 +1,49 @@
+import Service from '@ember/service';
+import { inject as service } from '@ember/service';
```

Comment:
Do we need to import both Service and inject ?

Reply:
Ah my oversight. Good catch!

Reply:
Actually yes. One is for ` Service.extend(` and the other is to inject `store: service('store'),`.